js/notify/electron-notify.html

```diff
@@ -2,11 +2,18 @@
 <head></head>
 <body style='margin:0; overflow: hidden; -webkit-user-select: none;'>
 <div id="container">
-	<div id="header">
-		<img src="" id="image" />
+    <div style="overflow: hidden; position: absolute; transform: translate(-25px, 10px);">
+        <img src="" id="symphony-logo">
+    </div>
+    <div id="header">
 		<span id="title"></span>
+        <!-- Hard coding this until the backend is ready -->
+        <span id="pod">symphony</span>
```

Comment:
see comment in jira ticket.  should we show at all if we don't have correct pod info.  check with product and design.